workflows/build.gradle

```diff
@@ -59,7 +59,9 @@ dependencies {
     // Token SDK dependencies.
     cordaCompile "$tokens_release_group:tokens-money:$tokens_release_version"
     cordaCompile "$tokens_release_group:tokens-workflows:$tokens_release_version"
+    cordapp "$tokens_release_group:tokens-selection:$tokens_release_version"
```

Comment:
In the Tokens SDK 1.2, `money` is now part of workflows

Reply:
Fixed